ci/pipeline.yaml

```diff
@@ -10,6 +10,14 @@ resource_types:
       username: ((harbor.username))
       password: ((harbor.token))
 
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: harbor-repo.vmware.com/dockerhub-proxy-cache/teliaoss/github-pr-resource
+      tag: latest
```

Comment:
I'd fix it to a known version.

Reply:
Any reason for not to fix it to a specific digest or tag? I wouldn't be great to get our CI to start failing because this external dependency changes

Reply:
My bad, I replied to this assuming that the PR was closed, sorry for the noise.

Reply:
Yep. Done.